dev_envs/mac-env-setup.md

```diff
@@ -249,14 +249,21 @@ and paste the contents of your clipboard
 There are some additional setup steps to access the
 [COOL (Cloud-Optimized Operations Lab)](https://github.com/cisagov/cool-system/wiki/):
 
-1. [Install MIT Kerberos](https://github.com/cisagov/cool-system/wiki/Installing-MIT-Kerberos)
+**Note:**
+These steps are only necessary if you want to manually configure these vs.
+using the Jamf-pushed configuration.
```

Comment:
This is a [specific GitHub-flavored Markdown syntax](https://github.com/orgs/community/discussions/16925) that I suggested. Also I would like consensus on if we prefer `vs` or `vs.` as #79 has the former and this has the latter (pre-suggestion).

```suggestion
> **Note**
> These steps are only necessary if you want to manually configure these vs
> using the Jamf-pushed configuration.
```

Reply:
The dot is grammatically correct for American English.

Reply:
Yes I prefer the dot but it was not mentioned in the other PR so I wanted to get a team consensus on preference.

Reply:
Team Dot over here!

Reply:
I think the three of us makes it an official consensus!

Reply:
I added the dot to @mcdonnnj's suggested change.

Reply:
@mcdonnnj - please see commits dc39701 and a1831ea.